doc/ROS2_PX4_Offboard_Tutorial.md

```diff
@@ -0,0 +1,415 @@
+[0.585s] WARNING:colcon.colcon_ros.prefix_path.catkin:The path '/home/analyst/microros_ws/install/micro_ros_msgs' in the environment variable CMAKE_PREFIX_PATH doesn't exist
+Starting >>> micro_ros_setup
+Finished <<< micro_ros_setup [1.52s]                  
+
+Summary: 1 package finished [1.80s]
+analyst@casa:~/microros_ws$ source install/local_setup.sh 
+analyst@casa:~/microros_ws$ ros2 run micro_ros_setup create_agent_ws.sh
+..
+=== ./uros/micro-ROS-Agent (git) ===
+Cloning into '.'...
+=== ./uros/micro_ros_msgs (git) ===
+Cloning into '.'...
+#All required rosdeps installed successfully
+analyst@casa:~/microros_ws$ ros2 run micro_ros_setup build_agent.sh
+Building micro-ROS Agent
+[0.593s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/analyst/microros_ws/install/micro_ros_agent' in the environment variable AMENT_PREFIX_PATH doesn't exist
+[0.593s] WARNING:colcon.colcon_ros.prefix_path.ament:The path '/home/analyst/microros_ws/install/micro_ros_msgs' in the environment variable AMENT_PREFIX_PATH doesn't exist
+[0.593s] WARNING:colcon.colcon_ros.prefix_path.catkin:The path '/home/analyst/microros_ws/install/micro_ros_agent' in the environment variable CMAKE_PREFIX_PATH doesn't exist
+[0.593s] WARNING:colcon.colcon_ros.prefix_path.catkin:The path '/home/analyst/microros_ws/install/micro_ros_msgs' in the environment variable CMAKE_PREFIX_PATH doesn't exist
+Starting >>> micro_ros_msgs
+Finished <<< micro_ros_msgs [5.51s]                     
+Starting >>> micro_ros_agent
+--- stderr: micro_ros_agent                                
+Cloning into 'xrceagent'...
+Switched to a new branch 'ros2'
+HEAD is now at 3eb56b5 Release v2.3.0
+CMake Warning (dev) at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
+  The package name passed to `find_package_handle_standard_args` (tinyxml2)
+  does not match the name of the calling package (TinyXML2).  This can lead
+  to problems in calling code that expects `find_package` result variables
+  (e.g., `_FOUND`) to follow a certain pattern.
+Call Stack (most recent call first):
+  cmake/modules/FindTinyXML2.cmake:40 (find_package_handle_standard_args)
+  /opt/ros/humble/share/fastrtps/cmake/fastrtps-config.cmake:51 (find_package)
+  CMakeLists.txt:153 (find_package)
+This warning is for project developers.  Use -Wno-dev to suppress it.
+
+---
+Finished <<< micro_ros_agent [23.2s]
+
+Summary: 2 packages finished [29.0s]
+  1 package had stderr output: micro_ros_agent
+
+```
+
+Try running the agent:
+
+```
+source install/local_setup.sh
+ros2 run micro_ros_agent micro_ros_agent udp4 --port 8888
+```
+
+It should respond with:
+
+```
+[1670101284.566587] info     | UDPv4AgentLinux.cpp | init                     | running...             | port: 8888
+[1670101284.566822] info     | Root.cpp           | set_verbose_level        | logger setup           | verbose_level: 4
+```
```

Comment:
Could we just link the instruction to an external link? building a micro-ros-agent is not really related to this package

Reply:
Done